### PR TITLE
Don't broadcast peer routes

### PR DESCRIPTION
**Pull request by michielbdejong**
Implements https://github.com/interledgerjs/ilp-routing/issues/29

### CODE REVIEW

File: src/lib/message-router.js [hunk 1]
```diff
@@ -132,7 +134,8 @@ MessageRouter.prototype.receiveRoutes = function * (payload, sender) {
     // We received a route from another connector, but that connector
     // doesn't actually belong to the route, so ignore it.
     if (route.source_account !== sender) continue
-    if (this.routingTables.addRoute(route)) gotNewRoute = true
+    const isPeerRoute = route.destination_ledger.startsWith(PEER_LEDGER_PREFIX)
+    if (this.routingTables.addRoute(route) && !isPeerRoute) gotNewRoute = true
```
Comment (sentientwaffle) [inline src/lib/message-router.js]: If RouteBroadcaster isn't going to broadcast the peer routes anyway, why is this needed?

Comment (michielbdejong) [inline src/lib/message-router.js]: ah, good point. I'll switch them around so the lazy evaluation ensures that routes *to* `peer.` ledgers never enter routing tables.
